client/lib/convert-url.js

```diff
@@ -1,23 +0,0 @@
-var QueryString = require('querystring');
```

Comment:
This file was use to convert url between a json and url format for the ajax request. We only use a html to json format url which is simplified with the file to-json-url.js